### DISCUSSION TITLE
does athena works in dimensionless units?

### DISCUSSION THREAD

**Discussion by Rogerod23** [PrincetonUniversity/athena#443] [Q&A] [answered]
Hello all. I just started to learn how Athena works and I noticed that there is no mention on the wiki, or in the problem generator files about the units on which the code is working, are the variables are defined in CGS, SI, dimensionless?. I know the main focus of the code is for astrophysics and that means it is probably on CGS units, but I just wanted to make sure because it is hard to understand it if one looks at the input files because everything (even the limits of the mesh) is defined with only numbers and there is no mention (comments) of the units is given. 

Thank you for your response,

-Rogelio.

**Comment by felker** [comment, marked as answer]
See here: https://github.com/PrincetonUniversity/athena/discussions/401#discussioncomment-1792317

**Reply by Rogerod23** [reply]
Thank you very much!
